packages/common/src/bag3d/common/utils/database.py

```diff
@@ -7,7 +7,7 @@
 
 
 def load_sql(filename: str = None,
-             query_params: dict = None):
+             query_params: dict = None): # pragma: no cover 
```

Comment:
this is only so that this is not taken into account for the coverage estimation - I have added it in many places. It can of course be removed.